src/Tribe/Views/V2/Hooks.php

```diff
@@ -566,6 +567,36 @@ public function manage_sensitive_info( $post ) {
 		}
 	}
 
+	/**
+	 * Fires to add JSON LD to Single Event
+	 *
+	 * @since TBD
+	 *
+	 */
```

Comment:
:no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: c4f6a43b4971440cd1d611b40156cf7d</sub>